platform/options.api/src/org/netbeans/modules/options/OptionsPanel.java

```diff
@@ -877,7 +869,7 @@ private void addCategoryButtons () {
                 
     private CategoryButton addButton (CategoryModel.Category category) {
         int index = buttons.size ();
-        CategoryButton button = isNimbus || isGTK 
+        CategoryButton button = UIManager.getBoolean("Nb.options.categories.button.useNimbusCategoryButton") //NOI18N
```

Comment:
Would be nicer to reintroduce the same constants as in LFCustoms at the top of the file again (now it can be not public)